src/test/java/spoon/test/comment/CommentTest.java

```diff
@@ -749,6 +749,7 @@ public void testDocumentationContract() throws Exception {
 		final Launcher launcher = new Launcher();
 		launcher.getEnvironment().setNoClasspath(true);
 		launcher.getEnvironment().setCommentEnabled(true);
+		launcher.getEnvironment().setComplianceLevel(10);
```

Comment:
why is this required here?

Reply:
I added a new snippet in the code with the following code:
```
var x = 10;
```

This won't compile if I do not use a compliance level to 10.